### DISCUSSION TITLE
Is there a way to get the `slick.jdbc.PositionedResult` if we already define the table?

### DISCUSSION THREAD

**Discussion by tanin47** [slick/slick#2942] [Questions] [answered]
For example, I already define the table here: 

```
class QuestionTable(tag: Tag) extends Table[Question](tag, "question") {
  def id = column[Long]("id", O.PrimaryKey, O.AutoInc)
  def question = column[String]("question")
  def * = (
    id, 
    question
  ).<>((Question.apply _).tupled, Question.unapply)
}
```

Now, for a certain reason, I have a need to write raw SQL:

```
db
      .run {
        sql"""
          SELECT
            id,
            question
          FROM "question"
        """.as[Question]
      }
```

`as[Question]` would miss the implicit view available from `slick.jdbc.PositionedResult`. I could write it up but I would prefer not to duplicate what is already coded in `QuestionTable`.

Is there a way to get `PositionedResult` from `QuestionTable`?

Thank you!

**Comment by nafg** [comment, marked as answer]
I don't think so.

How would it work?
